integrations/qdrant/src/haystack_integrations/document_stores/qdrant/document_store.py

```diff
@@ -84,6 +85,7 @@ def __init__(
         metadata: Optional[dict] = None,
         write_batch_size: int = 100,
         scroll_size: int = 10_000,
+        payload_field_to_index: Optional[List[dict]] = None,
```

Comment:
Wouldn't `payload_fields_to_index` be a more appropriate identifier?

Reply:
You are right